### PR TITLE
Fixes lint errors in kubeapiserver packages

### PR DESCRIPTION
**Pull request by cheftako**
**What this PR does / why we need it**:
Fixes lint errors in kubeapiserver/admission, kubeapiserver/authorizer,
kubeapiserver/authenticator. Also enables lint testing of these
directories.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes part of issues/68026
/kind bug

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/kubeapiserver/authorizer/config.go [hunk 1]
```diff
@@ -33,7 +33,8 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac/bootstrappolicy"
 )
 
-type AuthorizationConfig struct {
+//Config contains the data on how to authorize a request to the Kube API Server
```
Comment (sttts) [inline pkg/kubeapiserver/authorizer/config.go]: nit: space missing

**Comment by sttts** [comment]
lgtm
/approve

Needs rebase.

File: pkg/kubeapiserver/admission/config.go [hunk 2]
```diff
@@ -37,14 +37,15 @@ import (
 	quotainstall "k8s.io/kubernetes/pkg/quota/v1/install"
 )
 
-// AdmissionConfig holds the configuration for initializing the admission plugins
-type AdmissionConfig struct {
+// Config holds the configuration needed to for initialize the admission plugins
```
Comment (lavalamp) [inline pkg/kubeapiserver/admission/config.go]: nit: "needed to for"

**Pull request merged**